scripts/pre-push.py

```
@@ -0,0 +1,142 @@
+#!/usr/bin/env python3
+import sys
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)